go/libraries/doltcore/sqle/dtables/schema_conflicts_table.go

```diff
@@ -162,19 +163,47 @@ func newSchemaConflict(ctx context.Context, table string, baseRoot *doltdb.RootV
 	}
 	baseFKs, _ := fkc.KeysForTable(table)
 
-	base, err := getCreateTableStatement(table, baseSch, baseFKs, bs)
-	if err != nil {
-		return schemaConflict{}, err
+	var base string
+	if baseSch != nil {
+		var err error
+		base, err = getCreateTableStatement(table, baseSch, baseFKs, bs)
+		if err != nil {
+			return schemaConflict{}, err
+		}
+	} else {
+		base = "<deleted>"
```

Comment:
Need tests for new table contents in the three cases shown here

Reply:
Added.